reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -219,8 +219,8 @@ abstract class TableReporter : Reporter() {
     }
 
     data class ResolvableError(
-            val error: OrtIssue,
-            val resolutions: List<ErrorResolution>
+            private val error: OrtIssue,
+            private val resolutions: List<ErrorResolution>
```

Comment:
Now that you are refactoring this class, you could also rename it to `ResolvableIssue`.